src/App.tsx

```diff
@@ -16,35 +22,96 @@
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const handleReverse = () => {
+    setIsReversed(!isReversed);
+  };
+
+  const handleReset = () => {
```

Comment:
[CODE STYLE] - The function name 'handleReset' is appropriate, but to make it more consistent with other function names, consider renaming it to 'resetSortOptions'.